tools/depends/configure.in

```diff
@@ -91,6 +91,39 @@ passed_ldflags="$LDFLAGS"
 passed_cxxflags="$CXXFLAGS"
 platform_cc=gcc
 platform_cxx=g++
+
+case $build in
+  arm*-*linux-android*)
+    build_os="android"
```

Comment:
haha - yeah you are right - i basically c&p'ed the case from the $host var below ... you want me to remove it?